src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/S3Service.kt

```diff
@@ -24,8 +25,14 @@ class S3Service {
     val objectRequest = PutObjectRequest.builder()
       .bucket(bucket.bucketName)
       .key(key)
+      .apply {
+        if (bucketOwnerFullControl) {
+          acl(ObjectCannedACL.BUCKET_OWNER_FULL_CONTROL)
```

Comment:
Should this get a unit test?

Reply:
i actually changed the signature slightly - so you just pass the ACL directly. i can add a test to ensure it's appliied properly.